sensors/avl/sensorhub-driver-avl-911/src/test/java/org/sensorhub/impl/sensor/avl/TestAvlDriverFromFile.java

```diff
@@ -50,7 +50,7 @@ public void init() throws Exception
         
         MultipleFilesProviderConfig filesConf = new MultipleFilesProviderConfig();
         filesConf.dataFolder = "src/test/resources";
-        config.commSettings = filesConf;
+//        config.commSettings = filesConf;
```

Comment:
I think this change needs to be reverted

Reply:
7e5ba9e1